xdp-dump/tests/test-xdpdump.sh

```diff
@@ -190,6 +191,8 @@ test_capt_pcapng()
 
 test_capt_term()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
```

Comment:
You should be able to make this test pass with the older kernel, guess only lines 196-197 will probably need to be conditional.